.github/workflows/gradle-check.yml

```diff
@@ -77,6 +77,14 @@ jobs:
               * **URL:** ${{ env.workflow_url }}
               * **CommitID:** ${{ env.pr_from_sha }}
 
+      - name: Extract Test Failure
+        if: ${{ github.event_name == 'pull_request_target' && failure() }}
+        run: |
+            TEST_FAILURES=`curl -s "${{ env.workflow_url }}/testReport/api/json?tree=suites\[cases\[status,className,name\]\]" | jq -r '.. | objects | select(.status=="FAILED",.status=="REGRESSION") | (.className + "." +  .name)' | uniq`
```

Comment:
Maybe use `uniq -c` to get a count of failures?

Reply:
I wanted to display the actual failures intead.

Reply:
`-c` will give you both the failure and the count:

```
curl -s "https://build.ci.opensearch.org/job/gradle-check/6876/testReport/api/json?tree=suites\[cases\[status,className,name\]\]" | jq -r '.. | objects | select(.status=="FAILED",.status=="REGRESSION") | (.className + "." + .name)' | uniq -c
      2 org.opensearch.index.ShardIndexingPressureConcurrentExecutionTests.testCoordinatingPrimaryThreadedUpdateToShardLimitsAndRejections
      4 org.opensearch.index.mapper.GeoPointFieldMapperTests.testLatLonInArrayMoreThanThreeValues
```

Reply:
Added. Nice bash-fu @andrross :)